### DISCUSSION TITLE
[Question]: How to use the latest commit version in z3py?

### DISCUSSION THREAD

**Discussion by hagozaebii** [Z3Prover/z3#6817] [Q&A]
Hello,

I am currently using z3 as a python library(so-called z3py). What if I want to use z3py with the latest commit version? Because I installed z3py with `pip3 install z3-solver`, I do not know how to install the latest z3py which uses latest commit.
Thanks for reading.

**Comment by NikolajBjorner** [comment]
1. Navigate to https://github.com/Z3Prover/z3/releases/tag/Nightly
2. Download the wheel corresponding to your platform. There are the following:

 [z3_solver-4.12.3.0-py2.py3-none-macosx_10_16_x86_64.whl](https://github.com/Z3Prover/z3/releases/download/Nightly/z3_solver-4.12.3.0-py2.py3-none-macosx_10_16_x86_64.whl)
[z3_solver-4.12.3.0-py2.py3-none-macosx_11_0_arm64.whl](https://github.com/Z3Prover/z3/releases/download/Nightly/z3_solver-4.12.3.0-py2.py3-none-macosx_11_0_arm64.whl)
[z3_solver-4.12.3.0-py2.py3-none-manylinux2014_x86_64.whl](https://github.com/Z3Prover/z3/releases/download/Nightly/z3_solver-4.12.3.0-py2.py3-none-manylinux2014_x86_64.whl)
[z3_solver-4.12.3.0-py2.py3-none-win32.whl](https://github.com/Z3Prover/z3/releases/download/Nightly/z3_solver-4.12.3.0-py2.py3-none-win32.whl)
[z3_solver-4.12.3.0-py2.py3-none-win_amd64.whl](https://github.com/Z3Prover/z3/releases/download/Nightly/z3_solver-4.12.3.0-py2.py3-none-win_amd64.whl)


3. pip3 install 3_solver-4.12.3.0-py2.py3-none-win_amd64.whl or similar corresponding to platform.

**Reply by hagozaebii** [reply]
@NikolajBjorner Thanks for your help!
